packages/api/src/lib/stripe.ts

```diff
@@ -0,0 +1,49 @@
+import Stripe from "stripe"
+
+interface StripeAccountDetails {
+  publicKey: string
+  secretKey: string
+  webhookSecret: string
+}
+
+const stripeAccountsMap = new Map<string, StripeAccountDetails>()
```

Comment:
i think this is a bit sus. Maps are usually used when the quantity of elements are unknown. In this case, the amount of possible stripe keys exists. You should make an object, with the `as const` to ensure that the object is readonly, and therefore each key is fully autocompletable

Reply:
I agree. Fixed.